

try{
  let a;
  a.length();
  console.log("inside try");
  
} catch(e){
  console.log("inside catch");
  
}


console.log("hi there");
